rllib/tuned_examples/ppo/pendulum-ppo.yaml

```diff
@@ -17,7 +17,6 @@ pendulum-ppo:
         lr: 0.0003
         sgd_minibatch_size: 64
         observation_filter: MeanStdFilter
-        seed: 42
```

Comment:
This change is not directly related.
I propose it because seeding PPO and Pendulum should be 100% robust with respect to seeding.
This test would hide any missing robustness.